content/projects/morse-code/_index.md

```diff
@@ -43,6 +45,7 @@ Make sure that when you do this, your code remains clean and DRY. DRY - it's a t
 
 - https://en.wikipedia.org/wiki/Morse_code
 - https://morsecode.scphillips.com/translator.html
+- https://morsecode.world/international/translator.html
```

Comment:
Why do we need 2 translators? Rather just use the best one and remove the other link.